assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego

```diff
@@ -1,91 +1,103 @@
-                "keyActualValue": 	sprintf("'%s' does not use '--no-install-recommends' flag to avoid installing additional packages", [runCmd.Original])
-              }
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("FROM={{%s}}.{{%s}}", [name, runCmd.Original]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("'%s' uses '--no-install-recommends' flag to avoid installing additional packages", [runCmd.Original]),
+		"keyActualValue": sprintf("'%s' does not use '--no-install-recommends' flag to avoid installing additional packages", [runCmd.Original]),
+	}
 }
 
-CxPolicy [ result ] {
-  runCmd := input.document[i].command[name][_]
-  isRunCmd(runCmd) 
-  
-  value := runCmd.Value
-  count(value) > 1 #command is in several tokens
-  
-  aptGetIdx := getAptGetIdx(value)
-  aptGetIdx != -1
-  aptGetCmdLastIdx := getCmdLastIdx(value,aptGetIdx)
+CxPolicy[result] {
+	runCmd := input.document[i].command[name][_]
+	isRunCmd(runCmd)
+
+	value := runCmd.Value
+	count(value) > 1 #command is in several tokens
+
+	aptGetIdx := getAptGetIdx(value)
+	aptGetIdx != -1
+	aptGetCmdLastIdx := getCmdLastIdx(value, aptGetIdx)
 
-  not checkRecommendsFlag(value,aptGetIdx,aptGetCmdLastIdx)
+	not checkRecommendsFlag(value, aptGetIdx, aptGetCmdLastIdx)
 
-  cmdFormatted := replace(runCmd.Original,"\"","'")
+	cmdFormatted := replace(runCmd.Original, "\"", "'")
 
 	result := {
-                "documentId": 		input.document[i].id,
-                "searchKey": 	    sprintf("FROM={{%s}}.{{%s}}", [name,runCmd.Original]),
-                "issueType":		"IncorrectValue", 
-                "keyExpectedValue": sprintf("'%s' uses '--no-install-recommends' flag to avoid installing additional packages", [cmdFormatted]),
-                "keyActualValue": 	sprintf("'%s' does not use '--no-install-recommends' flag to avoid installing additional packages", [cmdFormatted])
-              }
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("FROM={{%s}}.{{%s}}", [name, runCmd.Original]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("'%s' uses '--no-install-recommends' flag to avoid installing additional packages", [cmdFormatted]),
+		"keyActualValue": sprintf("'%s' does not use '--no-install-recommends' flag to avoid installing additional packages", [cmdFormatted]),
+	}
 }
 
-isRunCmd(com) = true{
-  com.Cmd == "run"
-} else = false
+isRunCmd(com) {
+	com.Cmd == "run"
+} else = false {
+	true
+}
 
 trimCmdEnd(cmd) = trimmed {
-  termOps := ["&&","||","|","&",";"]
-
-  splitStr := split(cmd," ")
-  some i, j
-      splitStr[i] == termOps[j] 
-      indexTerm := indexof(cmd,termOps[j])
-      trimmed := substring(cmd,0,count(cmd) - indexTerm)
-} else = cmd
+	termOps := ["&&", "||", "|", "&", ";"]
+
+	splitStr := split(cmd, " ")
+	some i, j
+	splitStr[i] == termOps[j]
+	indexTerm := indexof(cmd, termOps[j])
+	trimmed := substring(cmd, 0, count(cmd) - indexTerm)
```

Comment:
Should remove this indentation?

Reply:
indentations do not affect some instruction.. I believe that this some instruction could be removed in this case.